### DISCUSSION TITLE
Community usability testing

### DISCUSSION THREAD

**Discussion by Be-ing** [audacity/audacity#879] [Near future]
@clarissalimab has recently worked with several different projects to create [templates](https://github.com/clarissalimab/ux) for contributors to organize and document usability tests. Watch her [presentation](https://framatube.org/videos/watch/02151d79-d422-4012-a547-e01c9e54c7f7) about this from Libreplanet 2021. Would this be something that would be welcome for Audacity?

cc @Tantacrul

**Comment by Tantacrul** [comment]
Oh! Very nice :)

We'll be doing a lot of this. One of my favourite parts of the job.
